src/nrnoc/fadvance.cpp

```diff
@@ -507,9 +524,11 @@ void* nrn_fixed_step_lastpart(NrnThread* nth) {
 	nrn_daq_ai();
 #endif
 	fixed_record_continuous(nth);
-	CTADD
-	nrn_deliver_events(nth) ; /* up to but not past texit */
-	return nullptr;
+    CTADD {
```

Comment:
```suggestion
    CTADD
    {
```

Reply:
yeah ok - clang-format did that, I think?